desktop/settings/linux-settings.md

```diff
@@ -0,0 +1,151 @@
+available on the host machine. To increase processing power, set this to a
+higher number; to decrease, lower the number.
+
+- **Memory**. By default, Docker Desktop is set to use 25% of your host's memory. To increase the RAM, set this to a higher number. To decrease it, lower the number.
+
+- **Swap**. Configure swap file size as needed. The default is 1 GB.
+
+- **Disk image size**. Specify the size of the disk image.
+
+- **Disk image location**. Specify the location of the Linux volume where containers and images are stored.
+
+You can also move the disk image to a different location. If you attempt to move a disk image to a location that already has one, you are asked if you want to use the existing image or replace it.
+
+### File sharing
+
+Use File sharing to allow local directories on the Linux host to be shared with Linux containers.
+This is especially useful for
+editing source code in an IDE on the host while running and testing the code in a container.
+By default the `/home/<user>` directory is shared. If your project is outside this directory then it must be added
+to the list. Otherwise you may get `Mounts denied` or `cannot start service` errors at runtime.
+
+File share settings are:
+
+- **Add a Directory**: Click `+` and navigate to the directory you want to add.
+
+- **Remove a Directory**: Click `-` next to the directory you want to remove
+
+- **Apply & Restart** makes the directory available to containers using Docker's
+  bind mount (`-v`) feature.
+
+> Tips on shared folders, permissions, and volume mounts
+>
+ * Share only the directories that you need with the container. File sharing
+ introduces overhead as any changes to the files on the host need to be notified
+ to the Linux VM. Sharing too many files can lead to high CPU load and slow
+ filesystem performance.
+>
+ * Shared folders are designed to allow application code to be edited 
+ on the host while being executed in containers. For non-code items
+ such as cache directories or databases, the performance will be much 
+ better if they are stored in the Linux VM, using a [data volume](../../storage/volumes.md)
+ (named volume) or [data container](../../storage/volumes.md).
+
+### Proxies
+
+To configure HTTP proxies, switch on the **Manual proxy configuration** setting.
+
+Your proxy settings, however, are not propagated into the containers you start.
+If you wish to set the proxy settings for your containers, you need to define
+environment variables for them, just like you would do on Linux, for example:
+
+```console
+$ docker run -e HTTP_PROXY=http://proxy.example.com:3128 alpine env
+
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+HOSTNAME=b7edf988b2b5
+TERM=xterm
+HOME=/root
+HTTP_PROXY=http://proxy.example.com:3128
+```
+
+For more information on configuring the Docker CLI to automatically set proxy variables for both `docker run` and `docker build`
+see [Configure the Docker client](/network/proxy#configure-the-docker-client).
```

Comment:
This should probably be a relative link to the markdown file to allow IDEs to verify the link and anchor.